errbot/config-template.py

```diff
@@ -109,6 +109,9 @@
 # the user running Err.
 #AUTOINSTALL_DEPS = True
 
+# To use your own custom log formatter, uncomment and set BOT_LOG_FORMATTER
+# BOT_LOG_FORMATTER =
```

Comment:
Could include a link to the Python docs which explain the formatter syntax here? This would help tremendously with people wanting to use their own format who aren't sure how to specify it and/or where to find a reference of what is supported.